vtm.js

```diff
@@ -292,7 +443,6 @@ app.component('stat-category', {
     </div>`,
 });
 
-
```

Comment:
mam pocit, ze som videl niekde doporucenie, ze velke celky je lepsie oddelovat dvoma novymi riadkami a nie len jednym :-)

Reply:
ja neviem, jak to zabera viac jak jednu obrazovku tak je asi podla mna dobre oddelit to aspon tym jednym... ja neviem aku mame presne syntaxu a toto by sa asi nemuselo riesit u viacerych suborov. jako ja som zvykly oddelovat jednym ale ked je to komponenta tak by to mohli byt 2
ja som to asi zmazal lebo sa mi zdalo ze som tam nejak vytvoril prazdny riadok kde nebol